### PR TITLE
Upgrade kafka to 2.4.1

### PR DESCRIPTION
**Pull request by elonazoulay**

### CODE REVIEW

**Comment by kokosing** [comment]
Automation fails with:
```
[INFO] --- maven-dependency-plugin:3.1.1:analyze-only (default) @ tempto-kafka ---
[WARNING] Unused declared dependencies found:
[WARNING]    org.apache.kafka:kafka_2.12:jar:2.4.1:compile
[INFO] --------------------------------------------------------
```

File: pom.xml [hunk 1]
```diff
@@ -190,6 +191,30 @@
...
+                <groupId>io.netty</groupId>
+                <artifactId>netty-transport</artifactId>
+                <version>${dep.netty.version}</version>
+            </dependency>
+
+            <dependency>
+                <groupId>commons-cli</groupId>
+                <artifactId>commons-cli</artifactId>
+                <version>1.4</version>
+            </dependency>
```
Comment (kokosing) [inline pom.xml]: Why you need above dependencies? I don't see how they are related to kafka update.

Comment (elonazoulay) [reply pom.xml]: Fixed, commons-cli needs to be 1.4 due to enforcer error.

**Comment by elonazoulay** [comment]
FYI, tested presto product tests locally using this version and I no longer get the class not found exceptions (for presto pr #3941). Once this looks ok and lands I will update that pr to use tempto 180.

File: pom.xml [hunk 2]
```diff
@@ -190,6 +191,24 @@
...
+            <dependency>
+                <groupId>io.netty</groupId>
+                <artifactId>netty-buffer</artifactId>
+                <version>${dep.netty.version}</version>
+            </dependency>
+
+            <dependency>
+                <groupId>io.netty</groupId>
+                <artifactId>netty-transport</artifactId>
+                <version>${dep.netty.version}</version>
```
Comment (kokosing) [inline pom.xml]: Why do you need netty? I don't see it used anywhere. No other pom.xml (modules) is using that.

**Review by kokosing — APPROVED** [review]
% comment about netty

**Comment by wendigo** [comment]
I didn't check if there is a PR for that and I've made https://github.com/prestosql/tempto/pull/56 which was merged.

**Comment by wendigo** [comment]
Sorry @elonazoulay

File: pom.xml (hunk 2 above)
Comment (elonazoulay) [reply pom.xml]: Now that kafka_2_12 is removed it's not needed, thanks!

**Comment by elonazoulay** [comment]
> Sorry @elonazoulay

Hi @wendigo , no need to be sorry, you fixed the issue:)

**Comment by elonazoulay** [comment]
@kokosing , @wendigo - just some minor changes now. Let me know what you think.

**Review by wendigo — APPROVED** [review]
LGTM. Thanks!

**Comment by kokosing** [comment]
Merged, thanks!

**Pull request merged**